docs/en/setup/plugins/plugin-list.md

```diff
@@ -8,9 +8,9 @@
 - Fetcher
 - Filter
 - Forwarder
+	- [meter-grpc-forwarder](./forwarder_meter-grpc-forwarder.md)
 	- [nativelog-grpc-forwarder](./forwarder_nativelog-grpc-forwarder.md)
 	- [nativelog-kafka-forwarder](./forwarder_nativelog-kafka-forwarder.md)
-	- [meter-grpc-forwarder](./forwarder_meter-grpc-forwarder.md)
```

Comment:
Why remove this?

Reply:
this is auto-generated, I would check the reason

Reply:
only sequence change, order by letter